src/app/containers/Metadata/index.test.jsx

```diff
@@ -8,6 +8,12 @@ import services from '#server/utilities/serviceConfigs';
 import { RequestContextProvider } from '#contexts/RequestContext';
 import frontPageData from '#data/igbo/frontpage/index.json';
 import liveRadioPageData from '#data/korean/bbc_korean_radio/liveradio.json';
+import onClient from '#lib/utilities/onClient';
+
+let isOnClient = true;
+
+jest.mock('#lib/utilities/onClient', () => jest.fn());
+onClient.mockImplementation(() => isOnClient);
```

Comment:
Instead of mocking this function is it possible to do `global.window = undefined` in the no-js test?

This would remove this test's dependency on a utility function.

Reply:
I don't think it is possible!